### PR TITLE
zh-translation:/docs/examples/microservices-istio/setup-local-computer/index.md

### PR DESCRIPTION
**Pull request by TomatoAres**
ref: https://github.com/servicemesher/istio-official-translation/issues/1229

### CODE REVIEW

File: content/zh/docs/examples/microservices-istio/setup-local-computer/index.md [hunk 1]
```diff
@@ -1,50 +1,46 @@
...
 
 {{< boilerplate work-in-progress >}}
 
-In this module you prepare your local computer for the tutorial.
+在本模块中，您将为教程准备本地计算机
 
-1.  On your local computer, locate the `${NAMESPACE}-user-config.yaml` file you
-    created earlier in the tutorial, where `${NAMESPACE}` is the name of your
-    namespace. For example `tutorial-user-config.yaml`.
+1.  开始之前，在本地计算机上，创建你的 namespace：`${NAMESPACE}`,然后创建文件：`${NAMESPACE}-user-config.yaml`，`${NAMESPACE}` 是你之前创建的 namespace。例如 `tutorial-user-config.yaml` 。
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]:
```suggestion
1.  开始之前，在本地计算机上，创建你的 namespace：`${NAMESPACE}`，然后创建文件：`${NAMESPACE}-user-config.yaml`，`${NAMESPACE}` 是你之前创建的 namespace。例如 `tutorial-user-config.yaml` 。
```

File: content/zh/docs/examples/microservices-istio/setup-local-computer/index.md [hunk 2]
```diff
@@ -1,50 +1,46 @@
...
 
-1.  Install [Docker](https://docs.docker.com/install/).
+4.  安装 [Docker](https://docs.docker.com/install/)。
 
-1.  Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+5.  安装 [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)。
 
-1.  Set the `KUBECONFIG` environment variable for the `${NAMESPACE}-user-config.yaml`
-    configuration file:
+6.  设置环境变量 `KUBECONFIG`  为 配置文件 `${NAMESPACE}-user-config.yaml`：
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]:
```suggestion
6.  为配置文件 `${NAMESPACE}-user-config.yaml` 设置 `KUBECONFIG` 环境变量：
```

File: content/zh/docs/examples/microservices-istio/setup-local-computer/index.md [hunk 3]
```diff
@@ -1,50 +1,46 @@
...
 
     {{< text bash >}}
     $ export KUBECONFIG=./${NAMESPACE}-user-config.yaml
     {{< /text >}}
 
-1.  Verify that the configuration took effect by printing the current namespace:
+7.  通过打印当前名称空间来验证配置是否生效:
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]:
```suggestion
7.  通过打印当前命名空间来验证配置是否生效：
```

File: content/zh/docs/examples/microservices-istio/setup-local-computer/index.md [hunk 4]
```diff
@@ -1,50 +1,46 @@
...
 
     {{< text bash >}}
     $ kubectl config view -o jsonpath="{.contexts[?(@.name==\"$(kubectl config current-context)\")].context.namespace}"
     tutorial
     {{< /text >}}
 
+    您应该在输出中看到之前配置的名称空间的名称。
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]:
```suggestion
    您应该在输出中看到之前配置的命名空间的名称。
```

File: content/zh/docs/examples/microservices-istio/setup-local-computer/index.md [hunk 5]
```diff
@@ -1,50 +1,46 @@
...
     You should see the name of your namespace in the output.
```
Comment (yuxiaobo96) [inline content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]: 35 行英文未删除

File: content/zh/docs/examples/microservices-istio/setup-local-computer/index.md (hunk 2 above)
Comment (TomatoAres) [reply content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]: 结合下边的命令，我认为之前的翻译更合适一些

Comment (yuxiaobo96) [reply content/zh/docs/examples/microservices-istio/setup-local-computer/index.md]: 是的

**Comment by tanjunchen** [comment]
@TomatoAres  fix conflicts.

**Comment by yuxiaobo96** [comment]
@rootsongjc 
Please help merge.

**Pull request merged**